instrumentation/finatra-2.9/finatra-2.9.gradle

```diff
@@ -36,5 +42,10 @@ dependencies {
   // Required for older versions of finatra on JDKs >= 11
   testImplementation group: 'com.sun.activation', name: 'javax.activation', version: '1.2.0'
 
-  latestDepTestLibrary group: 'com.twitter', name: 'finatra-http_2.11', version: '20.6.+'
+  // TODO latestDepTestLibrary doesn't work here
```

Comment:
To confirm, this is because finatra 20.8.+ requires a new test class?

Reply:
i'm not sure why `latestDepTestLibrary` doesn't work here, just guessing it has something to do with your TODO markers above(?)

> TODO(anuraaga): Something about library configuration doesn't work well with scala compilation here.